docs/orangelight-figgy.md

```diff
@@ -0,0 +1,25 @@
+In the record page we build in the DOM an element with class 'document-thumbnail' where we also include the bibid in a data-attribute:
+```
+<div class="document-thumbnail" data-bib-id="99106565893506421">
+  <div class="default"></div>
+</div>
+```
+
+1. If the specific record id exists as a scanned resource in figgy:
+Orangelight uses the bibid from the data attribute and invokes an async graphql call against figgy.
+   * For better performance we filter the resource that has no FileSet.
```

Comment:
Does this mean the graphql query includes resources with or without FileSets?

Reply:
@sandbergja yes. We filter the results in the FiggyManifestManager. https://github.com/pulibrary/orangelight/blob/201c272894c1582c432d7eb0fc19e99373db1b29/app/javascript/orangelight/figgy_manifest_manager.js#L119